web/src/admin/AdminNode.js

```diff
@@ -444,10 +444,17 @@ class AdminNode extends React.Component {
     let value, data;
     switch (item) {
       case "node":
-        value = this.getIndexFromNodeId(this.state.form.parentNode);
-        data= this.state.nodes.map((node, i) => {
-          return {text: `${node.nodeInfo.name} / ${node.nodeInfo.id}`, id: i};
-        });
+        if (this.state.form.parentNode){
+          value = this.getIndexFromNodeId(this.state.form.parentNode);
+        }
+        if (this.state.nodes){
```

Comment:
Necessary?

Reply:
Actually on my local frontend test, if I remove them, still got the same error like #98.

Reply:
I mean `if (this.state.nodes){`

Reply:
If remove `if (this.state.nodes)` and no node in db,then:
![err](https://user-images.githubusercontent.com/46670806/102958507-ff9d5f00-4517-11eb-8039-70a77b1db43d.png)
The `this.state.nodes` is initialized to an array at the beginning,but somehow be set to null or undefined while page rendering(maybe in function `getNodes`)

Reply:
getNodes() should return [] for empty. Please fix at getNodes() side.